src/reaper_osara.cpp

```diff
@@ -1713,6 +1713,48 @@ void cmdMoveItems(Command* command) {
 		reportActionName(command->gaccel.accel.cmd);
 }
 
+void cmdMoveItemEdge(Command* command) {
+	MediaItem* item;
+	// try to provide information based on the last item spoken by osara if it is selected
+	if (currentItem && ValidatePtr((void*)currentItem, "MediaItem*")
+		&& IsMediaItemSelected(currentItem)
+	) 
+	item = currentItem;
+	else if(CountSelectedMediaItems(0)>0)
+		item = GetSelectedMediaItem(0, 0);
+	else {
+		outputMessage("no items selected");
+		return;
```

Comment:
I guess it's true that these commands always do nothing when no items are selected. Even so, I'm uncomfortable with not passing the command to REAPER. If this assumption or code is wrong in some way and the command does actually do something, this will completely prevent users from being able to do it. Of course, if they do it, it'll be reported incorrectly, but I think that's better than not passing the command at all. We can still return here; just pass the command to REAPER first.